--- atividades/0104GEO001/0104GEO00102.js	2012-06-22 14:49:51.823885445 -0300
+++ atividades/0104GEO001/0104GEO00102.js.new	2012-06-22 15:07:35.939901434 -0300
@@ -445,7 +445,7 @@
 
 
 
-	PreloadImages('../imagens/logo.png','Tocantins.gif','Rondonia.gif','Acre.gif','rodape.png');
+	PreloadImages('../imagens/logo.png','tocantins.gif','rondonia.gif','acre.gif','rodape.png');
 
 
 	SetUpItems(ShuffleQs,QsToShow);
