--- atividades/0104GEO001/0104GEO00105.js	2012-06-22 15:07:38.399901471 -0300
+++ atividades/0104GEO001/0104GEO00105.js.new	2012-06-22 15:07:38.403901471 -0300
@@ -424,7 +424,7 @@
 
 var CorrectIndicator = ':-)';
 var IncorrectIndicator = 'X';
-var YourScoreIs = '<br>A sua pontua&#x00E7;&#x00E3;o &#x00E9;: ';
+var YourScoreIs = '<br>A sua pontua&ccedil;&atilde;o &eacute;: ';
 var CorrectResponse = 'MUITO BEM!';
 var IncorrectResponse = 'TENTE NOVAMENTE';
 var TotalUnfixedLeftItems = 0;
