--- atividades/0403POR004/0403POR00401.js	2012-06-22 15:07:46.879901596 -0300
+++ atividades/0403POR004/0403POR00401.js.new	2012-06-22 15:07:46.879901596 -0300
@@ -477,10 +477,10 @@
 var Feedback = '';
 var AcrossCaption = '';
 var DownCaption = '';
-var Correct = 'Parab&#x00E9;ns. Bom trabalho!';
-var Incorrect = 'Algumas respostas est&#x00E3;o incorretas. Tente outra vez!'; 
+var Correct = 'Parab&eacute;ns. Bom trabalho!';
+var Incorrect = 'Algumas respostas est&atilde;o incorretas. Tente outra vez!'; 
 var GiveHint = 'Adicionada uma letra correta';
-var YourScoreIs = '<br>A sua pontua&#x00E7;&#x00E3;o &#x00E9;: ';
+var YourScoreIs = '<br>A sua pontua&ccedil;&atilde;o &eacute;: ';
 var BuiltGrid = '';
 var BuiltExercise = '';
 var Penalties = 0;
