--- atividades/0604GEO034/0604GEO03401.js	2012-06-22 15:07:52.931901688 -0300
+++ atividades/0604GEO034/0604GEO03401.js.new	2012-06-22 15:07:52.931901688 -0300
@@ -326,18 +326,18 @@
 switch(giorno) {
 case 0: nome = "Domingo"; break;
 case 1: nome = "Segunda-feira"; break;
-case 2: nome = "Terça-feira"; break;
+case 2: nome = "Ter&ccedil;a-feira"; break;
 case 3: nome = "Quarta-feira"; break;
 case 4: nome = "Quinta-feira"; break;
 case 5: nome = "Sexta-feira"; break;
-case 6: nome = "Sábado"; break;
+case 6: nome = "S&aacute;bado"; break;
 }
 strData = nome+", "+obj.getDate();
 mese = obj.getMonth();
 switch(mese) {
 case 0: nome = "Janeiro";  break;
 case 1: nome = "Fevereiro"; break;
-case 2: nome = "Março"; break;
+case 2: nome = "Mar&ccedil;o"; break;
 case 3: nome = "Abril"; break;
 case 4: nome = "Maio"; break;
 case 5: nome = "Junho"; break;
@@ -797,7 +797,7 @@
 }
 }
 else {
-window.alert("Aviso: impossível calcular o resultado final");
+window.alert("Aviso: imposs&iacute;vel calcular o resultado final");
 computeMarkErr = 1;
 voto = 0;
 }
@@ -955,11 +955,11 @@
 }
 function DoneBefore()
 {
-window.alert("Ops!Você já respondeu à pergunta");
+window.alert("Ops!Voc&ecirc; j&aacute; respondeu &agrave; pergunta");
 }
 function IncompletedQuest()
 {
-window.alert("Resposta incompleta. Por favor, responda a todas as opções");
+window.alert("Resposta incompleta. Por favor, responda a todas as op&ccedil;&otilde;es");
 }
 function Retire()
 {
@@ -985,7 +985,7 @@
 if (ns>1) msg+=ns+" Incorreto  .";
 if (ns==1) msg+=" apenas um incorreto.";
 if (ns==0) msg+=" nada incorreto.";
-msg+=" Você deve responder a ";
+msg+=" Voc&ecirc; deve responder a ";
 if (nr>1) msg+=nr+" perguntas.";
 else msg+=" um pergunta.";
 window.status=msg;
@@ -1094,23 +1094,23 @@
 else if (valid[i]==-1)
 documento.writeln("Resposta incorreta<BR>");
 else if (valid[i]==2)
-documento.writeln("Pergunta não validada<BR>");
+documento.writeln("Pergunta n&atilde;o validada<BR>");
 else if (valid[i]==3) {
 if (typeOfQuest[i]==2)
 documento.writeln("Respostas corretas "+nScore[i]+" em "+maxScore[i]+"<BR>");
 else if (typeOfQuest[i]==4)
 documento.writeln("descubra as palavras "+nScore[i]+" em "+maxScore[i]+"<BR>");
 else if (typeOfQuest[i]==5)
-documento.writeln("Fazer a correspondência "+nScore[i]+" em "+maxScore[i]+"<BR>");
+documento.writeln("Fazer a correspond&ecirc;ncia "+nScore[i]+" em "+maxScore[i]+"<BR>");
 }
 else if (valid[i]==5) {
-documento.write("pontuações "+nScore[i]);
+documento.write("pontua&ccedil;&otilde;es "+nScore[i]);
 if (groups[i]==0)
-documento.writeln(" (Pontuação "+maxScore[i]+")<BR>");
+documento.writeln(" (Pontua&ccedil;&atilde;o "+maxScore[i]+")<BR>");
 else if (groups[i]==1)
-documento.writeln("<BR>Resultado adicionado à pontuação anterior<BR>");
+documento.writeln("<BR>Resultado adicionado &agrave; pontua&ccedil;&atilde;o anterior<BR>");
 else if (groups[i]==2)
-documento.writeln("<BR>Pontuação (cálculos das respostas às perguntas seguintes): "+maxScore[i]+"<BR>");
+documento.writeln("<BR>Pontua&ccedil;&atilde;o (c&aacute;lculos das respostas &agrave;s perguntas seguintes): "+maxScore[i]+"<BR>");
 }
 documento.writeln("</I>");
 if (valid[i]==2) {
@@ -1327,18 +1327,18 @@
 }
 if (end_test==1) {
 window.status = "";
-window.alert("Já respondeu a todas as perguntas");
+window.alert("J&aacute; respondeu a todas as perguntas");
 EndQuiz();
 }
 return end_test;
 }
 function PrintOpenAnswer(n,outMsg) {
-outMsg.value = "Questão respondida";
+outMsg.value = "Quest&atilde;o respondida";
 end_test = CountAnswers();
 PrintStatusBar();
 if (end_test==1) {
 window.status = "";
-window.alert("Já respondeu a todas as perguntas");
+window.alert("J&aacute; respondeu a todas as perguntas");
 EndQuiz();
 }
 return end_test;
@@ -1366,7 +1366,7 @@
 textField.value = "Resposta selecionada";
 if (end_test==1) {
 window.status = "";
-window.alert("Já respondeu a todas as perguntas");
+window.alert("J&aacute; respondeu a todas as perguntas");
 EndQuiz();
 }
 return end_test;
@@ -1378,9 +1378,9 @@
 end_test = CountAnswers();
 PrintStatusBar();
 if (silent==0) {
-msg = "Pontuação : " + nScore[n];
+msg = "Pontua&ccedil;&atilde;o : " + nScore[n];
 if (groups[n]==0) 
-msg += " (Pontuação "+maxScore[n]+")";
+msg += " (Pontua&ccedil;&atilde;o "+maxScore[n]+")";
 textField.value = msg;
 if (getGuessAns(n)<getNumAns(n))
 msg += "<BR>A resposta correta era " + getListOfRightAns(n);
@@ -1392,7 +1392,7 @@
 textField.value = "Resposta selecionada";
 if (end_test==1) {
 window.status = "";
-window.alert("Já respondeu a todas as perguntas");
+window.alert("J&aacute; respondeu a todas as perguntas");
 EndQuiz();
 }
 return end_test;
@@ -1415,8 +1415,8 @@
 document.writeln (bodyTagFrame1);
 document.writeln("<CENTER><FONT SIZE=6 FACE='Arial' COLOR='#FF0000'>Cuidado!</FONT><BR>");
 document.writeln ("<FONT SIZE=+2>Password incorreta</FONT><BR>");
-document.writeln ("Não pode avançar no quiz<BR>");
-document.writeln ("Para mais informação, contacte o autor do quiz<BR>");
+document.writeln ("N&atilde;o pode avan&ccedil;ar no quiz<BR>");
+document.writeln ("Para mais informa&ccedil;&atilde;o, contacte o autor do quiz<BR>");
 document.writeln("<P><INPUT TYPE='BUTTON' VALUE='FECHAR' OnClick='top.close()'>");
 document.writeln("</CENTER>");
 document.writeln ("</body></html>");
@@ -1430,9 +1430,9 @@
 PrintOpenHTML(document,"",1);
 document.writeln (bodyTagFrame1);
 document.writeln("<CENTER><BR><BR><FONT SIZE=6 FACE='Arial' COLOR='#FF0000'>Cuidado!</FONT><BR>");
-document.writeln ("<FONT SIZE=+2>É impossível reiniciar o quiz</FONT><BR>");
-document.writeln ("É impossível iniciar o quiz<BR>");
-document.writeln ("Para mais informação, contacte o autor do quiz<BR>");
+document.writeln ("<FONT SIZE=+2>&Eacute; imposs&iacute;vel reiniciar o quiz</FONT><BR>");
+document.writeln ("&Eacute; imposs&iacute;vel iniciar o quiz<BR>");
+document.writeln ("Para mais informa&ccedil;&atilde;o, contacte o autor do quiz<BR>");
 document.writeln("<P><INPUT TYPE='BUTTON' VALUE='FECHAR' OnClick='top.close()'>");
 document.writeln("</CENTER>");
 document.writeln ("</body></html>");
@@ -1609,7 +1609,7 @@
   doc.writeln("<TABLE cellpadding=5 border=0 width=100%><TR><TD>");
   doc.writeln(""+quizTitle+"");
   doc.writeln("</TD></TR></TABLE></TD></TR>");
-  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Número de perguntas</TD>");
+  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;N&uacute;mero de perguntas</TD>");
   doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;"+(questions-invisibleQuests)+"</TD></TR>");
   if (identityName!="") {
   doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Nome</TD>");
@@ -1619,7 +1619,7 @@
   doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;"+nc+"</TD></TR>");
   doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Respostas incorretas</TD>");
   doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;"+ns+"</TD></TR>");
-  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Perguntas não respondidas</TD>");
+  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Perguntas n&atilde;o respondidas</TD>");
   doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;"+falseQuests+"</TD></TR>");
   if (maxtime>0) {
   doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Tempo de preenchimento</TD>");
@@ -1628,11 +1628,11 @@
   doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Data</TD>");
   doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;"+DataToStringa()+"</TD></TR>");
   if (computeMarkErr==0) {
-  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Pontuação</TD>");
+  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Pontua&ccedil;&atilde;o</TD>");
   doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;"+GetVoto(voto)+"</TD></TR>");
   } else {
-  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Pontuação</TD>");
-  doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;sem pontuação</TD></TR>");
+  doc.writeln("<TR><TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;Pontua&ccedil;&atilde;o</TD>");
+  doc.writeln("    <TD bgcolor='#FFFFFF' nowrap>&nbsp;&nbsp;sem pontua&ccedil;&atilde;o</TD></TR>");
   }
   doc.writeln("</TABLE></TD></TR></TABLE></TD></TABLE></center>");
 }
@@ -1671,16 +1671,16 @@
 }
 function initShortQuestion() {
  qstHead[0]="EM QUAL PLANETA MORAMOS?";
- qstHead[1]="NO NOSSO PLANETA HÁ MAIS:";
- qstHead[2]="QUAL O NOME DO PAÍS ONDE VOCÊ MORA?";
+ qstHead[1]="NO NOSSO PLANETA H&Aacute; MAIS:";
+ qstHead[2]="QUAL O NOME DO PA&Iacute;S ONDE VOC&Ecirc; MORA?";
  qstHead[3]="QUAL O NOME DA CAPITAL DO BRASIL?";
  qstHead[4]="AS ILHAS E CONTINENTES FORMAM A PARTE:";
- qstHead[5]="O BRASIL ESTÁ LOCALIZADO...";
- qstHead[6]="VOCÊ ESTUDA EM QUAL CIDADE?";
+ qstHead[5]="O BRASIL EST&Aacute; LOCALIZADO...";
+ qstHead[6]="VOC&Ecirc; ESTUDA EM QUAL CIDADE?";
  qstHead[7]="QUAL O NOME DO OCEANO QUE BANHA O BRASIL?";
- qstHead[8]="DOIS PAÍSES DA AMÉRICA DO SUL NÃO FAZEM FRONTEIRA COM O BRASIL. QUAIS SÃO ELES?";
+ qstHead[8]="DOIS PA&Iacute;SES DA AM&Eacute;RICA DO SUL N&Atilde;O FAZEM FRONTEIRA COM O BRASIL. QUAIS S&Atilde;O ELES?";
  qstHead[9]="UM FERIADO NACIONAL OCORRE:";
- qstHead[10]="SE PUDESSE ESCOLHER, EM QUE PAÍS GOSTARIA DE MORAR? POR QUÊ?";
+ qstHead[10]="SE PUDESSE ESCOLHER, EM QUE PA&Iacute;S GOSTARIA DE MORAR? POR QU&Ecirc;?";
 }
 
 
