--- atividades/0802GEO002/0802GEO00201b.js	2012-06-22 15:08:04.495901862 -0300
+++ atividades/0802GEO002/0802GEO00201b.js.new	2012-06-22 15:08:04.495901862 -0300
@@ -481,11 +481,11 @@
 }
 
 var Feedback = '';
-var Correct = '<img src="imagens/ok.gif" alt="ok.gif" title="ok" width="45" height="48"></img></br>Parab&#x00E9;ns!!! ';
-var Incorrect = '<img src="imagens/no.gif" alt="no.gif" title="no" width="50" height="50"></img></br>Algumas respostas est&#x00E3;o incorretas. Tente outra vez!'; 
+var Correct = '<img src="imagens/ok.gif" alt="ok.gif" title="ok" width="45" height="48"></img></br>Parab&eacute;ns!!! ';
+var Incorrect = '<img src="imagens/no.gif" alt="no.gif" title="no" width="50" height="50"></img></br>Algumas respostas est&atilde;o incorretas. Tente outra vez!'; 
 var GiveHint = 'Foi adicionada a letra seguinte:';
 var CaseSensitive = false;
-var YourScoreIs = '</br>A sua pontua&#x00E7;&#x00E3;o &#x00E9;';
+var YourScoreIs = '</br>A sua pontua&ccedil;&atilde;o &eacute;';
 var Finished = false;
 var Locked = false;
 var Score = 0;
@@ -656,10 +656,10 @@
 
 function StartUp(){
 
-//*********** Acrescentado pelos Orientadores de Inform&#225;tica de Mogi das Cruzes ************
+//*********** Acrescentado pelos Orientadores de Inform&aacute;tica de Mogi das Cruzes ************
 	document.onkeyup = AvancaRetorna; //captura a tecla digitada
 	document.onmousedown=clicar; //captura o clique do mouse
-	document.oncontextmenu = mensagem;//bloqueia o bot&#227;o direito no Firefox
+	document.oncontextmenu = mensagem;//bloqueia o bot&atilde;o direito no Firefox
 //**** Fim ******
 
 	RemoveBottomNavBarForIE();
@@ -955,10 +955,10 @@
 
 
 
-//Fun&#231;&#227;o que permite o avan&#231;o e retorno de p&#225;ginas da atividade, sem que a mesma seja completada
-//Adicionada pelos Orientadores de Inform&#225;tica da PMMC
+//Fun&ccedil;&atilde;o que permite o avan&ccedil;o e retorno de p&aacute;ginas da atividade, sem que a mesma seja completada
+//Adicionada pelos Orientadores de Inform&aacute;tica da PMMC
 function AvancaRetorna(e) {
-	if (!e) e = window.event; //necess&#225;rio para validar a tecla no IE
+	if (!e) e = window.event; //necess&aacute;rio para validar a tecla no IE
 	letra = String.fromCharCode(e.keyCode);
 	btn_alt = e['altKey'];
 	btn_ctrl = e['ctrlKey'];
@@ -974,7 +974,7 @@
 	}
 }
 
-function clicar() { //fun&#231;&#227;o respons&#225;vel pelo tratamento do bot&#227;o direito do mouse no IE
+function clicar() { //fun&ccedil;&atilde;o respons&aacute;vel pelo tratamento do bot&atilde;o direito do mouse no IE
     if (event.button==2) {
      		mensagem;
     }
