--- atividades/0803POR017/0803POR01702.js	2012-06-22 15:08:37.923902363 -0300
+++ atividades/0803POR017/0803POR01702.js.new	2012-06-22 15:08:37.927902365 -0300
@@ -481,11 +481,11 @@
 }
 
 var Feedback = '';
-var Correct = 'Parab&#x00E9;ns!';
-var Incorrect = 'Algumas respostas est&#x00E3;o incorretas. Tente outra vez!'; 
+var Correct = 'Parab&eacute;ns!';
+var Incorrect = 'Algumas respostas est&atilde;o incorretas. Tente outra vez!'; 
 var GiveHint = 'Foi adicionada a seguinte letra:';
 var CaseSensitive = false;
-var YourScoreIs = 'Sua pontua&#x00E7;&#x00E3;o &#x00E9;: ';
+var YourScoreIs = 'Sua pontua&ccedil;&atilde;o &eacute;: ';
 var Finished = false;
 var Locked = false;
 var Score = 0;
