--- atividades/0804CIE001/0804CIE00101.js	2012-06-22 15:08:43.351902448 -0300
+++ atividades/0804CIE001/0804CIE00101.js.new	2012-06-22 15:08:43.351902448 -0300
@@ -636,9 +636,9 @@
 
 //JMATCH-SPECIFIC CORE JAVASCRIPT CODE
 
-var CorrectResponse = '<img src="imagens/ok.gif" alt="ok.gif" title="ok" width="45" height="48"></img></br>Parab&#x00E9;ns!!! ';
-var IncorrectResponse = '<img src="imagens/no.gif" alt="no.gif" title="no" width="50" height="50"></img></br>Algumas respostas est&#x00E3;o incorretas. Tente outra vez!';
-var YourScoreIs = '<br>A sua pontua&#x00E7;&#x00E3;o &#x00E9;: ';
+var CorrectResponse = '<img src="imagens/ok.gif" alt="ok.gif" title="ok" width="45" height="48"></img></br>Parab&eacute;ns!!! ';
+var IncorrectResponse = '<img src="imagens/no.gif" alt="no.gif" title="no" width="50" height="50"></img></br>Algumas respostas est&atilde;o incorretas. Tente outra vez!';
+var YourScoreIs = '<br>A sua pontua&ccedil;&atilde;o &eacute;: ';
 var DivWidth = 600; //default value
 var FeedbackWidth = 200; //default
 var ExBGColor = '#e2e2e2';
@@ -819,10 +819,10 @@
 
 F = new Array();
 F[0] = new Array();
-F[0][0]='Vis&#x00E3;o';
+F[0][0]='Vis&atilde;o';
 F[0][1] = 1;
 F[1] = new Array();
-F[1][0]='Audi&#x00E7;&#x00E3;o';
+F[1][0]='Audi&ccedil;&atilde;o';
 F[1][1] = 2;
 F[2] = new Array();
 F[2][0]='Tato';
@@ -849,7 +849,7 @@
 D[2][1] = 3;
 D[2][2] = 0;
 D[3] = new Array();
-D[3][0]='L&#x00ED;ngua';
+D[3][0]='L&iacute;ngua';
 D[3][1] = 4;
 D[3][2] = 0;
 D[4] = new Array();
