--- atividades/1003MAT003/1003MAT00304.js	2012-06-22 15:09:36.495903246 -0300
+++ atividades/1003MAT003/1003MAT00304.js.new	2012-06-22 15:09:36.495903246 -0300
@@ -606,9 +606,9 @@
 
 //JMATCH-SPECIFIC CORE JAVASCRIPT CODE
 
-var CorrectResponse = 'Correto. Parab&#x00E9;ns!';
-var IncorrectResponse = 'Algumas respostas est&#x00E3;o incorretas...tente novamente';
-var YourScoreIs = '<br>A sua pontua&#x00E7;&#x00E3;o &#x00E9;: ';
+var CorrectResponse = 'Correto. Parab&eacute;ns!';
+var IncorrectResponse = 'Algumas respostas est&atilde;o incorretas...tente novamente';
+var YourScoreIs = '<br>A sua pontua&ccedil;&atilde;o &eacute;: ';
 var DivWidth = 600; //default value
 var FeedbackWidth = 200; //default
 var ExBGColor = '#ffffff';
